sct.py

```diff
@@ -97,7 +97,7 @@
     send_perf_email
 from sdcm.parallel_timeline_report.generate_pt_report import ParallelTimelinesReportGenerator
 from sdcm.utils.aws_utils import AwsArchType
-from sdcm.utils.gce_utils import SUPPORTED_PROJECTS
+from sdcm.utils.gce_utils import SUPPORTED_PROJECTS, gce_public_addresses
```

Comment:
It is update of what is done in the previous commit of this PR.
So, would be really good to reduce such unneeded changes in scope of the single PR.

Reply:
it just a mistake, a fix that was apply into the wrong commit.
I'll move it into the right place

Reply:
fixed